Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM golang:1.12-alpine as builder
-ENV DESIRED_VERSION=v3.4.0
+ENV DESIRED_VERSION=v3.4.2
```

Comment:
Looks like this line is not used any more. The latest version of Helm is installed with `wget -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3`.
```suggestion
```

Reply:
it is used [here](https://github.com/helm/helm/blob/ba9cad9379f33d909bff8e85a85c935ab727482b/scripts/get#L90)